tests/test_v1_api.py

```diff
@@ -830,3 +830,117 @@ def test_adding_few_systems(self):
 
         for system_id in system_ids:
             self.assertIn(system_id, response_system_ids)
+
+
```

Comment:
Thank you so much for adding tests.  You set a good example for us all, myself included.

Reply:
It can be much better....

But thank you :-)